odk1-src/central-install-digital-ocean.rst

```diff
@@ -0,0 +1,235 @@
+
+   .. image:: /img/central-install/access-page.png
+
+Once you are in your server, you'll want to change your password so that people snooping your email do not gain access. Type ``passwd`` and press **Enter**, then follow the instructions to choose a new password. From now on, you will use that password to log in.
+
+Now you'll need to download the software. In the server window, type ``git clone https://github.com/opendatakit/central`` and press **Enter**. It should think for some time and download many things. Then type `cd central` to start working with the software.
+
+   .. image:: /img/central-install/cloned.png
+
+You now have the framework of the server software, but some components are missing. Type ``git submodule update -i`` and press **Enter** to download them.
+
+Next, you need to update some settings. Type ``nano .env`` and press **Enter**. This will launch a text editing application.
+
+ - Change the ``SSLTYPE`` line to read: ``SSLTYPE=letsencrypt``. This instructs the server to attempt to obtain a security certificate from the free Let's Encrypt provider.
+ - Change the ``SSLDOMAIN`` line so that after the ``=`` is the domain name you registered above. As an example: ``SSLDOMAIN=MyOdkCollectionServer.com``. Do not include anything like ``http://``.
+ - Change the ``SSLEMAIL`` line so that after the ``=`` is your own email address. The Let's Encrypt service will use this address only to notify you if something is wrong with your security certificate.
+ - Hold ``Ctrl`` and press ``x`` to quit the text editor. Press ``y`` to indicate that you want to save the file, and then press **Enter** to confirm the file name. Do not change the file name.
+
+   .. image:: /img/central-install/nano.png
+
+Now, we will bundle everything together into a server. Type ``docker-compose build`` and press **Enter** to do this. This will take a long time and generate quite a lot of text output. Don't worry if it seems to pause without saying anything for a while. When it finishes, you should see some "Successfully built" type text and get your input prompt back. When that happens, type ``docker-compose create`` and press **Enter**.
+
+Once that is complete, congratulations! You have installed your copy of ODK Central. Next, we need to teach the server how to start it up, and do so.
+
+.. _central-install-digital-ocean-startup:
+
+Starting up ODK Central
+-----------------------
+
+Now, we want to run your new ODK server software. But we don't want to only run it once: if we do that, then if your server machine crashes or restarts, the software won't start back up. We want to tell the machine to always run the server. To teach it to do this, we have to copy a file to the right spot. To do this, run ``cp files/docker-compose@.service /etc/systemd/system``.
```

Comment:
"if your machine crashes or restarts". I think using server there is confusing since you say machine and server later.